src/framework/statusDropdown/StatusDropdown.jsx

```diff
@@ -114,7 +121,11 @@ export default class StatusDropdown extends Component {
       this.optionToMetaDataMap[this.props.selectedOption].labelClass;
     const labelClasses = classNames('statusDropdownLabel', {
       [additionalLabelClass]: additionalLabelClass,
+      'statusDropdownLabel--disabled':
+        this.props.selectedOption === StatusDropdown.OPTIONS.DISABLED,
     });
+    const isReadonly =
+      this.props.selectedOption === StatusDropdown.OPTIONS.DISABLED;
```

Comment:
we should declare this before labelClasses, so we can call isReadonly instead of doing the comparison again